docs/Examples/client-api.js

```diff
@@ -105,12 +129,14 @@ async function callNodeApi() {
   console.log('Sent TX:');
   console.log(hash);
 
-  const tx = await wallet.getTX(hash);
+  // API call: walletClient.getTX('test', hash)
+  const tx = await walletClient.request('GET', `/wallet/test/tx/${hash}`);
 
   console.log('Sent TX details:');
   console.log(tx);
 
   await callNodeApi();
+  process.exit(0);
```

Comment:
Is it not closing already, is this necessary?

Reply:
I'll close node and wallet instead, makes for a better example